README.md

```diff
@@ -46,6 +46,17 @@ powershell
 Set-ExecutionPolicy unrestricted
 ```
 
+#### If build with PowerShell does not work
+If you get errors like "Error: Java heap space" or "dx tool failed", you probably have to allocate more Java memory:
+- open the jvm.config file inside your Flex Air SDK directory and set at least 512 Mb
```

Comment:
This file is not located in the base Flex SDK directory.  It is located in %FLEX_HOME%/bin

Probably should specify to look for the line that says this, for clarity, and specify what exactly this is doing.

Reply:
Yes, sorry it's in the "bin" directory, I'll correct that.